compiler/quilt/test/test_command.py

```diff
@@ -250,6 +250,12 @@ def mock_git_clone(cmd):
         assert hasattr(test, 'foo')
         assert isinstance(test.foo(), pd.DataFrame)
 
+    def test_build_yaml_syntax_error(self):
+        path = os.path.dirname(__file__)
+        buildfilepath = os.path.join(path, 'build_bad_syntax.yml')
+        with assertRaisesRegex(self, command.CommandException, 'Syntax error'):
+            command.build('user/test', buildfilepath)
+
```

Comment:
Can we add a unit test for a syntax error in the checks.yml file too?

Reply:
Definitely.

Reply:
Ok, that's been pushed.  Notes:

* I was previously working in command.py, and testing in command_test.py.
* Since the exception catch was moved to build.py, I made tests in build_test.py
* I left the command_test tests in, as they do test (slightly) different code.